public/index.html

```diff
@@ -52,9 +31,18 @@
 
 
 		</table>
+
+
+		<canvas id="myCanvas" width="650" height="650" style="border:1px solid #c3c3c3;">
```

Comment:
Jeg synes det er kanskje litt for mye "space" mellom linjene. Kan du fjerne dem?